drivers/volume/kdmp/kdmp.go

```diff
@@ -980,8 +978,8 @@ func GetGenericDriverName() string {
 	return storkvolume.KDMPDriverName
 }
 
-func (k *kdmp) getSnapshotClassName(backup *storkapi.ApplicationBackup) string {
-	if snapshotClassName, ok := backup.Spec.Options[optCSISnapshotClassName]; ok {
+func (k *kdmp) getSnapshotClassName(driver string, backup *storkapi.ApplicationBackup) string {
+	if snapshotClassName, ok := backup.Spec.CSISnapshotClassMap[driver]; ok {
 		return snapshotClassName
```

Comment:
If user has not selected anything it should be ""  irrespective of if default has or not. In case of CSI we are sending `default` or `Default` so we should be using that